packages/slice-machine/src/hooks/useSliceMachineConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { SliceMachineConfig } from "@slicemachine/manager";
+import { managerClient } from "@src/managerClient";
+
+import { useRequest } from "@prismicio/editor-support/Suspense";
+
+async function getSliceMachineConfig() {
+  try {
+    return await managerClient.project.getSliceMachineConfig();
+  } catch (e) {
+    console.error(e);
```

Comment:
Actually that's my bad here on Marc PR, but better to add something like:

```suggestion
    console.error("Error while trying to get Slice Machine config", e);
```